src/storm-pomdp/analysis/IterativePolicySearch.cpp

```diff
@@ -85,7 +85,7 @@ namespace storm {
         template <typename ValueType>
         bool IterativePolicySearch<ValueType>::initialize(uint64_t k) {
             STORM_LOG_INFO("Start intializing solver...");
-
+            bool delayedSwitching = false; // Notice that delayed switching is currently broken.
```

Comment:
Maybe elaborate what is broken?

Reply:
Done (I dont really know what is broken. I know what I want to do first in order to fix this ;-))